tests/test_eeasisss_integration.py

```
@@ -0,0 +1,48 @@
+import os
+
+import pytest
+
+from phaseshifts.wrappers import EEASiSSSWrapper
+
+
+def _has_eeasisss_lib():
```

Ruff (0.14.10)

Missing return type annotation for private function `_has_eeasisss_lib`

Add return type annotation: `bool`

(ANN202)